lib/legacy.coffee

```diff
@@ -1,3 +1,5 @@
+fs = require('fs')
```

Comment:
Why you would name a local variable to fs? It's a bad practice not the other way.